From 33aa64d97af374a8616765d6d26f4766c8998229 Mon Sep 17 00:00:00 2001
From: Ashwin Sekhar T K <ashwin.sekhar@cavium.com>
Date: Thu, 9 May 2019 04:59:56 -0700
Subject: [PATCH] Add object files src/ComputeProlongation.o
 src/ComputeRestriction.o

---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 55f0ea6..fba41ff 100644
--- a/Makefile
+++ b/Makefile
@@ -15,6 +15,7 @@ HPCG_DEPS = src/CG.o src/CG_ref.o src/TestCG.o src/ComputeResidual.o \
          src/ComputeDotProduct_ref.o src/finalize.o src/init.o src/mytimer.o src/ComputeSPMV.o \
          src/ComputeSPMV_ref.o src/ComputeSYMGS.o src/ComputeSYMGS_ref.o src/ComputeWAXPBY.o src/ComputeWAXPBY_ref.o \
          src/ComputeMG_ref.o src/ComputeMG.o src/ComputeProlongation_ref.o src/ComputeRestriction_ref.o src/GenerateCoarseProblem.o \
+         src/ComputeProlongation.o src/ComputeRestriction.o \
 	 src/ComputeOptimalShapeXYZ.o src/MixedBaseCounter.o src/CheckAspectRatio.o src/OutputFile.o
 
 bin/xhpcg: src/main.o $(HPCG_DEPS)
-- 
1.8.3.1
